.github/workflows/deploy-claw.yml

```diff
@@ -0,0 +1,48 @@
+name: "Deploy CLAW"
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'develop'
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout CLAW
+      uses: actions/checkout@v4
+    - name: Setup Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: '3.2' # Not needed with a .ruby-version file
```

Comment:
Can we just add the .ruby-version file instead of having this comment?

Reply:
Fixed it

Reply:
Done